src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceV2.java

```diff
@@ -0,0 +1,141 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import static uk.gov.hmcts.reform.sscs.transform.deserialize.SubmitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseDataV2;
+
+import feign.FeignException;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.UnaryOperator;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpStatus;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.config.CitizenCcdService;
+import uk.gov.hmcts.reform.sscs.domain.wrapper.SyaCaseWrapper;
+import uk.gov.hmcts.reform.sscs.exception.ApplicationErrorException;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
+import uk.gov.hmcts.reform.sscs.model.SaveCaseOperation;
+import uk.gov.hmcts.reform.sscs.model.SaveCaseResult;
+import uk.gov.hmcts.reform.sscs.model.draft.SessionDraft;
+import uk.gov.hmcts.reform.sscs.service.converter.ConvertAIntoBService;
+
+@Service
+@Slf4j
+@ConditionalOnProperty(name = "feature.submit-appeal-service-v2.enabled", havingValue = "true")
+public class SubmitAppealServiceV2 extends AbstractSubmitAppealService {
```

Comment:
Names having versioning isnt ideal again in terms of naming conventions, better to move it to package .v2 and have same name unless u think its going to help someone finding the file in IDE.

Reply:
Moved to v2 package.